README.md

```diff
@@ -73,7 +73,7 @@ Participants will get the most from this lesson if they have familiarity with:
 >
 > _For example:_
 >
-> *   Do you want to make changes to your theme yourself?
```

Comment:
First I change it to 'make', but I think it must be plural, then I change it to 'makes' again.

Reply:
I believe that the singular form would be best here but I think you changed it back.

```suggestion
> *   Do you want to make changes to your theme yourself?
```

Reply:
OK, then I create new PR for it :)